src/liballoc/btree/map.rs

```diff
@@ -576,6 +576,35 @@ impl<K: Ord, V> BTreeMap<K, V> {
         }
     }
 
+    /// Returns a references to the key-value pair corresponding to the supplied
```

Comment:
Technically, it returns a pair which consists of references to the key and value - not sure if that's an important distinction, but we should remove " a " since it's not grammatically correct I think.

Reply:
Fixed - removed the part about references as that's included in the type signature anyway.